src/DependencyInjection/MercureExtension.php

```diff
@@ -107,6 +107,8 @@ public function load(array $configs, ContainerBuilder $container)
                         $container->register($tokenFactory, LcobucciFactory::class)
                             ->addArgument($hub['jwt']['secret'])
                             ->addArgument($hub['jwt']['algorithm'])
+                            ->addArgument(null)
+                            ->addArgument($hub['jwt']['passphrase'])
```

Comment:
should we throw if passphrase is provided by `symfony/mercure` version doesn't support passphrase? 🤔 

cc @dunglas

Reply:
Yes, we should throw or bump the minimum version.

Reply:
okay, let's wait for a release on `symfony/mercure`, then we can bump the minimum here.